docker-build

```diff
@@ -6,4 +6,11 @@ IMAGE_NAME=mender-convert
 
 MENDER_CLIENT_VERSION="2.0.0"
 
-docker build . -t ${IMAGE_NAME} --build-arg mender_client_version=${MENDER_CLIENT_VERSION}
+DOCKER_ARGS="--build-arg mender_client_version=${MENDER_CLIENT_VERSION}"
+
+if [ "$1" = "arm64" ]; then
```

Comment:
we are doing this toolchain mapping (arm64 -> aarch64-linux-gnu) here which I think is fine -- makes it more usable.

But then I think you should move the "default" toolchain from the Dockerfile to here as well.